### PR TITLE
[Semio] LMS Update

### PR DESCRIPTION
**Pull request by chrismbirmingham**
This rebase brings the LMS PR up to parity with the latest development.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -94,6 +104,7 @@ class App extends React.Component<Props, State> {
           <Route path="/tutorials" exact component={Tutorials} />
           <Route path="/scene/:sceneId" component={Root} />
           <Route path="/challenge/:challengeId" component={ChallengeRoot} />
+          <Route path="/curriculum" component={CurriculumPage} />
```
Comment (tcorbly) [inline src/App.tsx:107]: A link to this needs to be added somewhere so that users have a way to get to it

File: src/lms/CurriculumPage.tsx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+          }}
+        />
+
+        <StyledTabBar
+          tabs={tabs}
+          index={tabIndex}
+          onIndexChange={setTabIndex}
+          theme={theme}
+        />
+        {/* <AddBotballPluginButton $theme={theme}>
```
Comment (tcorbly) [inline src/lms/CurriculumPage.tsx:138]: Currently the main issue I'm seeing is that there isn't a way to connect the LMS to Google Classroom. It looks like the place where the plugin was intended to be added is commented out.

**Review by tcorbly — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (chrismbirmingham) [reply src/App.tsx:107]: Happy to add it, but I was thinking we shouldn't add the button and make it visible until it is ready for launch. Do you have a preference for how and where to show this link?

**Review by tcorbly — APPROVED** [review]
We will accept the LMS PR as is for now so that next steps for the Google Classroom plugin can be decided.

**Pull request merged**